tests/e2e/helpers/database.ts

```diff
@@ -345,6 +345,9 @@ export class DatabaseHelper {
     async acceptLicenseTerms(): Promise<void> {
         await t.maximizeWindow();
         await userAgreementDialog.acceptLicenseTerms();
+        if (await myRedisDatabasePage.AddRedisDatabase.aiChatMessage.exists) {
```

Comment:
can you add TODO here, that we need to remove it with the June release, since as I remember it will be available only during the next sprint